src/fluvio.rs

```diff
@@ -95,6 +97,33 @@ pub struct Fluvio {
     inner: Rc<NativeFluvio>,
 }
 
+#[wasm_bindgen]
+#[derive(Debug)]
+pub enum JsLevel {
+    Error = "Error",
+    Warn = "Warn",
+    Info = "Info",
+    Debug = "Debug",
+    Trace = "Trace",
+}
+
+impl Into<Option<Level>> for JsLevel {
```

Comment:
Hmm. Is there a benefit of `Option<Level>` over just `Level`?

Reply:
Because it's coming from JS which could be anything I made it into an option. Is there a better way to do it perhaps?

Reply:
I think that if it is something that can fail we can use `TryFrom<JsLevel> for Level` instead